.github/ISSUE_TEMPLATE.md

```diff
@@ -5,7 +5,11 @@ everything here and write out the request, providing as much context as you can.
 
 -->
 
-### What version of Go (`go version`) and `dep` (`git describe --tags`) are you using?
+### What version of dep are you using (`dep version`)?
+<!--
+  If not using release version what version of Go (`go version`)
+  and `dep` (`git describe --tags`) are you using?
```

Comment:
let's replace with "If you installed `dep` via `go get`, report the version instead with `cd $GOPATH/src/github.com/golang/dep && git describe --tags`